src/radon/index.js

```diff
@@ -1,761 +1,1181 @@
+  },
+  [TYPES.INTEGER]: {
+    absolute: [0x20, [TYPES.INTEGER]],
+    asBytes: [0x21, [TYPES.BYTES]],
+    asFloat: [0x22, [TYPES.FLOAT]],
+    asString: [0x23, [TYPES.STRING]],
+    greaterThan: [0x24, [TYPES.BOOLEAN]],
+    lessThan: [0x25, [TYPES.BOOLEAN]],
+    match: [0x26, [PSEUDOTYPES.ARGUMENT]],
+    modulo: [0x27, [TYPES.INTEGER]],
+    multiply: [0x28, [TYPES.INTEGER]],
+    negate: [0x29, [TYPES.INTEGER]],
+    power: [0x2a, [TYPES.INTEGER]],
+    reciprocal: [0x2b, [TYPES.FLOAT]],
+    sum: [0x2c, [TYPES.INTEGER]],
+  },
+  [TYPES.FLOAT]: {
+    absolute: [0x30, [TYPES.INTEGER]],
+    asBytes: [0x31, [TYPES.BYTES]],
+    asString: [0x32, [TYPES.STRING]],
+    ceiling: [0x33, [TYPES.INTEGER]],
+    graterThan: [0x34, [TYPES.BOOLEAN]],
+    floor: [0x35, [TYPES.INTEGER]],
+    lessThan: [0x36, [TYPES.BOOLEAN]],
+    modulo: [0x37, [TYPES.FLOAT]],
+    multiply: [0x38, [TYPES.FLOAT]],
+    negate: [0x39, [TYPES.FLOAT]],
+    power: [0x3a, [TYPES.FLOAT]],
+    reciprocal: [0x3b, [TYPES.FLOAT]],
+    round: [0x3c, [TYPES.INTEGER]],
+    sum: [0x3d, [TYPES.FLOAT]],
+    truncate: [0x3e, [TYPES.INTEGER]],
+  },
+  [TYPES.STRING]: {
+    asBytes: [0x40, [TYPES.BYTES]],
+    asFloat: [0x41, [TYPES.FLOAT]],
+    asInteger: [0x42, [TYPES.INTEGER]],
+    length: [0x43, [TYPES.INTEGER]],
+    match: [0x44, [PSEUDOTYPES.ARGUMENT]],
+    parseJSON: [0x45, [TYPES.BYTES]],
+    parseXML: [0x46, [TYPES.MAP]],
+    asBoolean: [0x47, [TYPES.BOOLEAN]],
+    toLowerCase: [0x48, [TYPES.STRING]],
+    toUpperCase: [0x49, [TYPES.STRING]],
+  },
+  [TYPES.ARRAY]: {
+    asBytes: [0x50, [TYPES.BYTES]],
+    count: [0x51, [TYPES.INTEGER]],
+    every: [0x52, [TYPES.BOOLEAN]],
+    filter: [0x53, [TYPES.ARRAY, PSEUDOTYPES.INNER]],
+    flatten: [0x54, [TYPES.ARRAY, PSEUDOTYPES.PASSTHROUGH]],
+    get: [0x55, [PSEUDOTYPES.INNER]],
+    map: [0x56, [PSEUDOTYPES.ARGUMENT]],
+    reduce: [0x57, [PSEUDOTYPES.INNER]],
+    some: [0x58, [TYPES.BOOLEAN]],
+    sort: [0x59, [TYPES.ARRAY, PSEUDOTYPES.INNER]],
+    take: [0x5a, [TYPES.ARRAY, PSEUDOTYPES.INNER]],
+  },
+  [TYPES.MAP]: {
+    entries: [0x60, [TYPES.ARRAY, TYPES.BYTES]],
+    get: [0x61, [PSEUDOTYPES.INNER]],
+    keys: [0x62, [TYPES.STRING]],
+    values: [0x63, [PSEUDOTYPES.INNER]],
+  },
+  [TYPES.BYTES]: {
+    asArray: [0x70, [TYPES.ARRAY, TYPES.BYTES]],
+    asBoolean: [0x71, [TYPES.BOOLEAN]],
+    asFloat: [0x72, [TYPES.FLOAT]],
+    asInteger: [0x73, [TYPES.INTEGER]],
+    asMap: [0x74, [TYPES.MAP, TYPES.BYTES]],
+    asString: [0x75, [TYPES.STRING]],
+    hash: [0x76, [TYPES.BYTES]],
+  },
+  [TYPES.RESULT]: {
+    get: [0x80, [PSEUDOTYPES.INNER]],
+    getOr: [0x81, [PSEUDOTYPES.INNER]],
+    isOk: [0x82, [TYPES.BOOLEAN]],
+  },
+}
+
+function updateMarkup(markup, value) {
+  if (markup.markup_type === 'select') {
+    const optionsList = markup.options.map(option => option.label)
+
+    const selectedOperatorInfoEntry = Object.entries(OPERATOR_INFOS).find(operatorInfoEntry => {
```

Comment:
Using find is inefficient, you should build a map on startup and just index it by `(name, optionsList)` to get the desired `operatorId`, and then use the `operatorId` to index `OPERATOR_INFOS`.